analyses/mwi/dryspells_trigger/exploration/test_arc2_processing.md

```diff
@@ -0,0 +1,96 @@
+## ARC2 processing
```

Comment:
I think that we can also remove this file now that we have `15_mwi_obs_trigger_monitoring.md` ?

Reply:
Yeah, but might just leave now anyway, find it useful for testing as I've been amending.